resources/teamcity/web/keyman-web-test.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+# Copyright (C) 2025 SIL International. All rights reserved.
+# Distributed under the MIT License. See LICENSE.md file in the project
+# root for full license information.
+#
+# TC build script for Keyman Web/Test
+
+# shellcheck disable=SC2164
+# shellcheck disable=SC1091
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../../../resources/build/builder.inc.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+# shellcheck disable=SC2154
+. "${KEYMAN_ROOT}/resources/teamcity/includes/tc-helpers.inc.sh"
+
+################################ Main script ################################
+
+builder_describe \
+  "Run tests for native KeymanWeb" \
+  "all            run all actions" \
+  "configure      install dependencies" \
+  "build          build Web + embedded" \
+  "test           run native KeymanWeb tests and check build size"
+
+builder_parse "$@"
+
+cd "${KEYMAN_ROOT}/web"
+
+function install_dependencies_action() {
+  if is_windows; then
```

Comment:
```suggestion
  if ! is_linux; then
```

This should only run on Linux, not on Windows or macOS